### PR TITLE
FRONTEND::Routes: Implemented Register route

### PR DESCRIPTION
**Pull request by Elohim2598**

### CODE REVIEW

**Review by j-eick — APPROVED** [review]
Looks good. I didn't check every line though.

File: frontend/src/routes/auth/register/+page.svelte
Comment (adan-ayaz-stan) [inline frontend/src/routes/auth/register/+page.svelte:1]: Read the following things:
1. https://learn.svelte.dev/tutorial/form-validation
2. https://svelte-forms-lib-sapper-docs.vercel.app/introduction

Should make it a lot easier and less of a headache but still a headache :D.

**Review by adan-ayaz-stan — CHANGES_REQUESTED** [review]

File: frontend/src/components/ui/button/Button.svelte [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	export let label: string;
+	export let type: 'button' | 'reset' | 'submit' = 'button';
+	export let disabled: boolean = false;
+	export let onClick: (event: MouseEvent) => void;
+	export let variant: string = 'primary';
+	export let size: string = 'edium';
```
Comment (j-eick) [inline frontend/src/components/ui/button/Button.svelte:7]: That's supposed to be **"medium"** I think.

**Review by j-eick — CHANGES_REQUESTED** [review]

**Review by j-eick — APPROVED** [review]
Looks good. 

Sidenote:
Whenever I use `flex justify-center items-center` I sometimes consider using `grid place-items-center` instead, as it's a bit less to type. Anyways...

**Pull request merged**